src/Stores/ArticleStore.js

```diff
@@ -0,0 +1,26 @@
+import { extendObservable } from "mobx";
+import ArticlesData from "./../Data/Articles.json";
+import moment from "moment";
+
+class ArticleStore {
+  constructor() {
+    extendObservable(this, {
+      articles: []
+    });
+  }
+
+  loadArticles = options => {
```

Comment:
I’d consider returning a Promise, even though this is sync for now

Reply:
Sure. I'll do that.